### PR TITLE
Make certification cancellation success page into React component

### PR DESCRIPTION
**Pull request by mmlumba**
Connects #1722

### CODE REVIEW

File: client/app/containers/BaseContainer.jsx [hunk 1]
```diff
@@ -17,17 +15,17 @@ import EstablishClaimComplete from './EstablishClaimPage/EstablishClaimComplete'
...
 const Pages = {
   CaseWorkerIndex,
+  CancelCertificationConfirmation,
   DecisionReviewer: Reader,
   Intake,
   EstablishClaim,
   EstablishClaimCancel,
   EstablishClaimComplete,
   StyleGuideIndex,
-  StyleGuideModal,
```
Comment (amprokop) [inline client/app/containers/BaseContainer.jsx]: Is this an unrelated change? Do we still need these?

File: client/app/components/StatusMessage.jsx [hunk 2]
```diff
@@ -7,12 +7,12 @@ export default class StatusMessage extends React.Component {
   render() {
     let {
       checklist,
-      // TODO(nth) This is not a good variable name. It shadows the classNames node module.
-      // And it's too generic – the classNames are applied to one specific child element, but
-      // you'd never know what that element is by looking at the variable name.
-      classNames,
+      checklistClassNames,
       example,
+      // leadMessageList: 20px font for page text: used for primary message text
```
Comment (amprokop) [inline client/app/components/StatusMessage.jsx]: Hmm — not sure of the purpose of this comment. Is there a reason to specify these pixel values here? Shouldn't that info be found elsewhere?

File: app/views/certification_cancellations/show.html.erb [hunk 3]
```diff
@@ -1,7 +1,4 @@
 <% content_for :page_title do certification_header("Not Certified") end %>
```
Comment (amprokop) [inline app/views/certification_cancellations/show.html.erb]: Do we still need this file at all— did you consider just using React Router to handle this?

See how we get to this erb file currently:
https://github.com/department-of-veterans-affairs/caseflow/blob/c0a6636f3c4ced94b33fcd5774452f1558bc5f9a/client/app/certification/CancelCertificationModal.jsx#L119

Here's an example of how to do it with react router:
https://github.com/department-of-veterans-affairs/caseflow/blob/c0a6636f3c4ced94b33fcd5774452f1558bc5f9a/client/app/certification/ConfirmHearing.jsx#L220

Here's an example of where we define routes:
https://github.com/department-of-veterans-affairs/caseflow/blob/c0a6636f3c4ced94b33fcd5774452f1558bc5f9a/client/app/certification/Certification.jsx#L181

All page routing should be handled by `Certification.jsx`and its router if possible, to keep things simple. What do you think?

File: client/app/components/StatusMessage.jsx (hunk 2 above)
Comment (mmlumba) [reply client/app/components/StatusMessage.jsx]: This is just to inform the developer what the differences between the body texts are in `StatusMessage` and which prop to use when implementing the component.

File: client/app/containers/BaseContainer.jsx (hunk 1 above)
Comment (mmlumba) [reply client/app/containers/BaseContainer.jsx]: Its unrelated, but these are still rendered correctly in Styleguide and `StyleGuideModal` and `StyleGuideTabs` don't need to be in BaseContainer

Comment (amprokop) [reply client/app/containers/BaseContainer.jsx]: sounds good

**Comment by amprokop** [comment]
@mmlumba — how did you test this? do you have screenshots of how the app works?

File: client/app/components/StatusMessage.jsx (hunk 2 above)
Comment (amprokop) [reply client/app/components/StatusMessage.jsx]: It's a little confusing/hard to understand for me — can we be more explicit? More like `leadMessageList is type X and is used for Y`.

**Comment by mmlumba** [comment]
@amprokop I used the regular test suite to see if the functionality works, and I confirmed that it saved the cancellation information in the `certification_cancellations` table. Screenshot with React info:
screen shot 2017-11-17 at 9 57 11 am

**Comment by mmlumba** [comment]
@amprokop Error message needs to be converted to React view also (right now its still rendered in Rails), but its your call if you want me to update that in another PR or update in the same one. The success page is already rendered in React correctly.
